src/index.ts

```diff
@@ -1,26 +1,65 @@
+import { Request, Response } from 'express'
 import { Reshuffle, BaseHttpConnector, EventConfiguration } from 'reshuffle-base-connector'
-import { Request, Response, NextFunction } from 'express'
 import mondaySdk from 'monday-sdk-js'
 
-const DEFAULT_WEBHOOK_PATH = '/monday-event'
+const WEBHOOK_PATH = '/reshuffle-monday-connector/webhook'
 
 export interface MondayConnectorConfigOptions {
   token: string
+  baseURL: string
 }
 
+const MONDAY_ID_REGEX = /^\d{9,10}$/
+
 export interface MondayConnectorEventOptions {
-  boardId: number
-  baseUrl: string
+  boardId: string | number
   type:
-    | 'incoming_notification'
-    | 'change_column_value'
-    | 'change_specific_column_value'
-    | 'create_item'
-    | 'create_update'
-  config?: Record<string, unknown>
-  path?: string
-  webhookId?: string
-  deleteWebhookOnExit?: boolean
+    | 'IncomingNotification'
+    | 'ChangeColumnValue'
+    | 'ChangeSpecificColumnValue'
+    | 'CreateItem'
+    | 'CreateUpdate'
+  columnId?: string // for type === ChangeSpecificColumnValue
+}
+
+const EventTypes = {
+  IncomingNotification: 'incoming_notification',
+  ChangeColumnValue: 'change_column_value',
+  ChangeSpecificColumnValue: 'change_specific_column_value',
+  CreateItem: 'create_item',
+  CreateUpdate: 'create_update',
+}
+
+const SepytTneve = {
+  incoming_notification: 'IncomingNotification',
+  change_column_value: 'ChangeColumnValue',
+  update_column_value: 'ChangeColumnValue',
+  change_specific_column_value: 'ChangeSpecificColumnValue',
+  update_specific_column_value: 'ChangeSpecificColumnValue',
```

Comment:
ChangeSpecificColumnValue => UpdateSpecificColumnValue

Reply:
Clarified in comments